apps/settings/test/unit/modules/bluetooth/bluetooth_device_test.js

```diff
@@ -35,6 +50,7 @@ suite('BluetoothDevice', function() {
         assert.equal(bluetoothDeviceCreated.paired, mockDeviceObj.paired);
         assert.equal(bluetoothDeviceCreated.address, mockDeviceObj.address);
         assert.equal(bluetoothDeviceCreated.cod, mockDeviceObj.cod);
```

Comment:
Remember to remove this assertion if you are going to remove cod from the device object.